content/release-notes/1.21.0.md

```diff
@@ -0,0 +1,16 @@
+---
+date: 2020-10-23
+linktitle: "1.21.0"
+title: "1.21.0"
+kubernetes: "1.17, 1.18, and 1.19"
+---
+
+{{<changes>}}
+* Support bundle files will now be saved under a subdirectory, so that it won't pollute the current directory when extracted.
+* Rolling back to a version with known invalid files will show a "Invalid files" modal first before deploying.
```

Comment:
```suggestion
* Rolling back to a version with known invalid files will show an "Invalid files" modal first before deploying.
```

Reply:
Maybe "confirmation modal"? Showing a modal seems kind of weird, like that doesn't solve anything. Maybe what you really mean is "rolling back to a version with known invalid files will require confirmation before deploying"

Reply:
```suggestion
* Rolling back to a version with known invalid files will require confirmation before deploying.
```